database/src/main/java/com/firebase/ui/database/FirebaseRecyclerAdapter.java

```diff
@@ -110,9 +110,25 @@ public void onChanged(EventType type, int index, int oldIndex) {
             public void onCancelled(DatabaseError databaseError) {
                 FirebaseRecyclerAdapter.this.onCancelled(databaseError);
             }
+
+            @Override
+            public void onReady() {
+                FirebaseRecyclerAdapter.this.onDataChanged();
```

Comment:
Yeah, I wasn't sure. In the context of `FirebaseArray` the name `onDataChanged()` is too close to the existing `onChanged()` name. But I like the name as a whole.

We could consider renaming `onChanged()` to `onChildChanged()` (keep its signature aside from the name-change). Since the interface isn't part of the public API (yet), we could do that in a minor version.